[2024-07-13T02:01:07.584+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-13T02:01:07.610+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: parent_dag.trigger_raw_load_accounts manual__2024-07-11T20:36:59.640614+00:00 [queued]>
[2024-07-13T02:01:07.622+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: parent_dag.trigger_raw_load_accounts manual__2024-07-11T20:36:59.640614+00:00 [queued]>
[2024-07-13T02:01:07.623+0530] {taskinstance.py:2306} INFO - Starting attempt 2 of 3
[2024-07-13T02:01:07.643+0530] {taskinstance.py:2330} INFO - Executing <Task(TriggerDagRunOperator): trigger_raw_load_accounts> on 2024-07-11 20:36:59.640614+00:00
[2024-07-13T02:01:07.650+0530] {standard_task_runner.py:63} INFO - Started process 411546 to run task
[2024-07-13T02:01:07.654+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'parent_dag', 'trigger_raw_load_accounts', 'manual__2024-07-11T20:36:59.640614+00:00', '--job-id', '1175', '--raw', '--subdir', 'DAGS_FOLDER/dag_parent.py', '--cfg-path', '/tmp/tmpncscn1bx']
[2024-07-13T02:01:07.656+0530] {standard_task_runner.py:91} INFO - Job 1175: Subtask trigger_raw_load_accounts
[2024-07-13T02:01:07.706+0530] {task_command.py:426} INFO - Running <TaskInstance: parent_dag.trigger_raw_load_accounts manual__2024-07-11T20:36:59.640614+00:00 [running]> on host kali
[2024-07-13T02:01:07.799+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='parent_dag' AIRFLOW_CTX_TASK_ID='trigger_raw_load_accounts' AIRFLOW_CTX_EXECUTION_DATE='2024-07-11T20:36:59.640614+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-11T20:36:59.640614+00:00'
[2024-07-13T02:01:07.802+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-13T02:01:07.874+0530] {trigger_dagrun.py:222} INFO - Waiting for raw_load_accounts on 2024-07-12 20:31:07.802793+00:00 to become allowed state ['success'] ...
[2024-07-13T02:01:52.747+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-13T02:01:52.791+0530] {process_utils.py:132} INFO - Sending 15 to group 411546. PIDs of all processes in the group: [411546]
[2024-07-13T02:01:52.791+0530] {process_utils.py:87} INFO - Sending the signal 15 to group 411546
[2024-07-13T02:01:52.797+0530] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-07-13T02:01:52.798+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-13T02:01:52.825+0530] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/operators/trigger_dagrun.py", line 228, in execute
    time.sleep(self.poke_interval)
  File "/home/kali/Desktop/projects/git/bank_data_processing/myenv/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 2613, in signal_handler
    raise AirflowTaskTerminated("Task received SIGTERM signal")
airflow.exceptions.AirflowTaskTerminated: Task received SIGTERM signal
[2024-07-13T02:01:52.851+0530] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=parent_dag, task_id=trigger_raw_load_accounts, run_id=manual__2024-07-11T20:36:59.640614+00:00, execution_date=20240711T203659, start_date=20240712T203107, end_date=20240712T203152
[2024-07-13T02:01:52.896+0530] {process_utils.py:80} INFO - Process psutil.Process(pid=411546, status='terminated', exitcode=2, started='02:01:07') (411546) terminated with exit code 2
